### ISSUE TITLE
[Bug?]: setup auth dbAuth contentLines.findLastIndex 

### ISSUE THREAD

**Issue by pi0neerpat** [redwoodjs/redwood#8249]
### What's not working?

`yarn rw setup auth dbAuth` is failing on a fresh redwood v5 app.

Occurs when selecting either y/n for WebAuthn support

```bash
✔ Enable WebAuthn support (TouchID/FaceID)? See https://redwoodjs.com/docs/auth/dbAuth#webAuthn … yes
↓ Setting up Auth from scratch
✔ Generating auth api side files...
  ✔ Successfully wrote file `./api/src/functions/auth.js`
  ✔ Successfully wrote file `./api/src/lib/auth.js`
✖ contentLines.findLastIndex is not a function
◼ Creating web/src/auth.js
◼ Updating Routes file...
◼ Adding auth config to GraphQL API...
◼ Adding required web packages...
◼ Adding required api packages...
◼ Installing packages...
◼ Adding SESSION_SECRET var to .env...
◼ One more thing...
contentLines.findLastIndex is not a function
```

### What's your environment? (If it applies)

```shell
System:
    OS: Linux 5.15 Ubuntu 20.04.5 LTS (Focal Fossa)
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 16.19.0 - /tmp/xfs-6f927a3a/node
    Yarn: 3.5.0 - /tmp/xfs-6f927a3a/yarn
  Browsers:
    Firefox: 112.0.2
  npmPackages:
    @redwoodjs/auth-dbauth-setup: 5.0.6 => 5.0.6 
    @redwoodjs/core: 5.0.6 => 5.0.6
```

### Are you interested in working on this?

- [ ] I'm interested in working on this

**Comment by cannikin** [comment]
I’m out this week but can look into it when I’m back! But real quick @jtoar does that look familiar from v5 changes?

**Comment by jtoar** [comment]
@cannikin yeah I swear we had a bug just like this. Thanks @pi0neerpat I'll try reproducing now

**Comment by jtoar** [comment]
@pi0neerpat I couldn't reproduce with a new redwood project. Just confirming this is a brand new Redwood project? I think where we had trouble before was with different platforms and carriage returns etc, so I'll try to reproduce this on Linux as well (thanks for doing yarn rw info!)

**Comment by Josh-Walker-GM** [comment]
Hey @pi0neerpat, I was able to reproduce but with one caveat - node version.

I'm using WSL on windows and was able to generate a new project with `create redwood-app` and then setup dbAuth without any issues. My node version was 18.

I noticed your `rw info` states you're running node 16? I created another new project - I had to use 18 because recent changes to `create redwood-app` means we do not support creating a project with a lower node version. I then switched to node 16 and tried to setup dbAuth and got the same error you did.

Is there any reason you're on node 16? Are you able to try node 18 and see if you get the same error?
